### PR TITLE
[Form] Add phpdoc to `RequestHandlerInterface::isFileUpload()` method

### PR DESCRIPTION
**Pull request by issei-m**
| Q             | A
| ------------- | ---
| Branch?       | 2.7
| Bug fix?      | no
| New feature?  | no 
| BC breaks?    | no
| Deprecations? | no 
| Tests pass?   | Not confirmed, but the changes are completely minor
| Fixed tickets | -
| License       | MIT
| Doc PR        | -

### CODE REVIEW

File: src/Symfony/Component/Form/RequestHandlerInterface.php [hunk 1]
```diff
@@ -27,7 +27,9 @@
     public function handleRequest(FormInterface $form, $request = null);
 
     /**
-     * @param mixed $data
+     * Returns true if the given data is used to upload the file.
```
Comment (fabpot) [inline src/Symfony/Component/Form/RequestHandlerInterface.php]: `Returns true if the given data is a file upload`?

**Review by fabpot — CHANGES_REQUESTED** [review]

File: src/Symfony/Component/Form/RequestHandlerInterface.php [hunk 2]
```diff
@@ -27,7 +27,9 @@
...
+     *
+     * @param mixed $data the form field data
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Form/RequestHandlerInterface.php]: uppercase "T" [The...]

**Comment by issei-m** [comment]
@fabpot @nicolas-grekas Thank you guys for your advice! and now I've fixed it :)

**Review by nicolas-grekas — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
Thank you @issei-m.

**Pull request closed without merging**